code/modules/mod/mod_actions.dm

```diff
@@ -49,8 +53,10 @@
 	else
 		mod.choose_deploy(usr)
 
+/* SKYRAT EDIT REMOVAL - MODsuit pAIs
```

Comment:
Cant you just make a new action and leave this  in, but replace the instance where it gives the action?